lib/proxy_util.js

```diff
@@ -129,6 +129,14 @@ exports.validateEmptyString = function (value) {
 };
   
 exports.getProxyFromEnv = function (isHttps = true) {
+  const getDefaultPortIfNotSet = (proxyFromEnv) => {
```

Comment:
Not sure about js conventions, but I don't see why we'd want this function not to be top level